metashare/api/models.py

```diff
@@ -140,12 +141,13 @@ def is_devhub_enabled(self):
             return None
```

Comment:
We might wanna change the return type of this function to a strict `Boolean` and make these `None`s `False`s instead. Does returning `null` in the serializer have value to you?